src/scheduler/site_code.c

```diff
@@ -3723,7 +3723,7 @@ check_for_cycle_interrupt(int do_logging)
 		sprintf(log_buffer, "Too early to short circuit (%ds elapsed, need %ds)",
 			(int)(time(NULL) - interrupted_cycle_start_time),
 			conf.min_intrptd_cycle_length);
-		schdlog(PBSEVENT_DEBUG2, PBS_EVENTCLASS_SERVER, LOG_DEBUG, __func__, log_buffer);
+		log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_SERVER, LOG_DEBUG, __func__, log_buffer);
```

Comment:
Can this be log_eventf and we can get rid of sprintf as well?

Reply:
Same as above, this is NAS code.  They can make the modification if they wish.